repository_service_tuf_worker/repository.py

```diff
@@ -128,6 +128,24 @@ def __init__(self):
     def _settings(self) -> Dynaconf:
         return get_repository_settings()
 
+    @property
+    def _online_key(self) -> Key:
+        key_dict = self._settings.get_fresh("ONLINE_KEY")
+        if key_dict is not None:
+            return Key.from_dict(key_dict.pop("keyid"), key_dict)
+        else:
+            root: Metadata[Root] = self._storage_backend.get(Root.type)
+            # All roles except root share the same one key and it doesn't
+            # matter from which role we will get the key.
+            keyid: str = root.signed.roles["timestamp"].keyids[0]
+            return root.signed.keys[keyid]
```

Comment:
Shouldn't we also set it to speed for the next usage?
I mean, should we have it in Redis?

Reply:
Yes, you are right.
I forgot I can do that here as well.